html/indexbase.html

```diff
@@ -220,8 +226,50 @@ <h3 class="panel-title">Legend</h3>
             trigger: "hover",
             placement: "auto"
         })
+    }
+}
 
+var wereRacksUpdated = false;
+function updateRacksOnce(statusObject) {
```

Comment:
I think tracking how many times the function was called is an external concern and should not be part of what this function does